profiling/sync_to_user

```diff
@@ -0,0 +1,13 @@
+GET https://api-sandbox.simple.org/api/v3/blood_pressures/sync
+Accept: application/json
+Content-Type: application/json
+Authorization: Bearer aa9fdd9072e8c1726e5382170b2ae815c607d8d6718b223d1b0a9a8c2e7f8f55
+X-USER-ID: 18fa33d3-4f18-476a-8d63-51406352ba13
+X-FACILITY-ID: 36266c5b-490f-490f-8472-b8759aac0ace
```

Comment:
Are these valid, longstanding SBX creds? Should we perhaps not include em in the repo? If handling these secrets is awkward, we can just XXXXXXX em out and add instructions to populate em into the readme.

Reply:
I thought about this a bit. Shall we change this to `profiling/sync_to_user.example` with all the credentials XXXXX'd out, and .gitignore `profiling/sync_to_user`?

Reply:
Makes sense.